assets/app/styles/_core.less

```diff
@@ -193,6 +193,10 @@ html, body {
         }
       }
     }
+    .route {
```

Comment:
could make it an actual h2 for now to avoid needing to respecify h2 styles, and just style `h2.route` to remove any unwanted padding/margins for now